docs/deploying.md

```diff
@@ -89,6 +89,8 @@ docker run -d -p 5000:5000 \
 	registry:2
 ```
 
+If the certificate issuer supplies you with an 'intermediate' certificate, such as Gandi, you need to combine your certificate with the intermediates to form a 'certificate bundle'. You can do this using the cat command: ```cat server.crt GandiStandardSSLCA2.pem > server.with-intermediate.crt```. You can then configure the registry to use your certificate bundle with the ```REGISTRY_HTTP_TLS_CERTIFICATE``` environment variable.
```

Comment:
Remove mention of third party service. Intermediate certificates are a common thing. Perhaps, a link to https://en.wikipedia.org/wiki/Intermediate_certificate_authorities would help here.